### DISCUSSION TITLE
版面分析训练，找不到配置文件

### DISCUSSION THREAD

**Discussion by wonders7796** [PaddlePaddle/PaddleOCR#13707] [Q&A] [answered]
在开源项目的configs文件夹下，找不到配置文件configs/picodet/legacy_model/application/layout_analysis/picodet_lcnet_x1_0_layout.yml

**Comment by GreatV** [comment, marked as answer]
老哥文档写得挺清楚的 https://paddlepaddle.github.io/PaddleOCR/ppstructure/model_train/train_layout.html#51

https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.5/configs/picodet/legacy_model/application/layout_analysis
